### ISSUE TITLE
WebApp:az webapp up fails with Node JS

### ISSUE THREAD

**Issue by ericis** [Azure/azure-cli#16200]
## Describe the bug

A simple Node.js application will not properly deploy with the CLI command.

**Command Name**

`az webapp up`

**Errors:**
```
Deployment status endpoint https://some-unique-app-name-here.scm.azurewebsites.net/api/deployments/latest returns malformed data. Retrying...
```

## To Reproduce:

See this ["hello world" tutorial](https://github.com/ourchitecture/panda/tree/main/src/tutorials/node/azure), specifically deploying the [Node.js web app](https://github.com/ourchitecture/panda/tree/main/src/tutorials/node/azure/app#tutorials-nodejs-on-azure), which uses `az webapp up`.

You can simulate this by mounting a simple Node.js app (like the "/app" directory in the repo above) in docker and running:

- `docker run --rm -it -v $(pwd):/app -w /app microsoft/azure-cli:latest`
- `az --version` _\*Note: this still warns us that there are 2 updates available, even from the "latest" docker image._
- `pip install --upgrade azure-cli` to make sure we have the latest
- `az login` to authenticate
- `az webapp up ...`

## Expected Behavior

1. If the AppService does not already exist (tested), it will create it (it does), upload the files (it does) and start the application successfully (it does not)
2. If the AppService already exists (tested), it will upload the files (it does) and start the application (it does not)

## Environment Summary

tl;dr = the docker image "microsoft/azure-cli:latest"

- Windows 10
- Windows running WSL 2
- Windows running Docker Desktop community 2.5 and WSL 2 integration enabled
- WSL 2 running Ubuntu 20 LTS with docker CLI installed and integrated with host docker from Windows
- Ubuntu running docker running the image "microsoft/azure-cli:latest"

    ```shell
    cat /etc/os-release
        NAME="Alpine Linux"
        ID=alpine
        VERSION_ID=3.11.6
        PRETTY_NAME="Alpine Linux v3.11"
        HOME_URL="https://alpinelinux.org/"
        BUG_REPORT_URL="https://bugs.alpinelinux.org/"
    ```

After upgrading the `az` command to latest...

    ```shell
    az --version
        azure-cli                         2.16.0

        core                              2.16.0
        telemetry                          1.0.6

        Python location '/usr/local/bin/python'
        Extensions directory '/root/.azure/cliextensions'

        Python (Linux) 3.6.10 (default, May 21 2020, 05:54:39) 
        [GCC 9.2.0]

        Legal docs and information: aka.ms/AzureCliLegal


        Your CLI is up-to-date.
    ```

**Comment by yungezz** [comment]
route to appropriate team

**Comment by panchagnula** [comment]
@calvinsID seems like the error here is the SCM endpoint query to deployment status it returning malformed data "Deployment status endpoint https://some-unique-app-name-here.scm.azurewebsites.net/api/deployments/latest returns malformed data. Retrying..."  may be @sanchitmehta can help us figure out why this happens during deployment.
While this is not a directly an issue with az webapp up - we can work on making the experience better for users by handling this exception better. Please let me know what you find. Thanks!

**Comment by StrawnSC** [comment]
@ericis -- thanks for reporting this. I'm unable to repro the issue here. Could you retry with the latest CLI version and let us know if you're still seeing the bug?

**Comment by ericis** [comment]
> @ericis -- thanks for reporting this. I'm unable to repro the issue here. Could you retry with the latest CLI version and let us know if you're still seeing the bug? 

Will do @StrawnSC

**Comment by seligj95** [comment]
please share if this is still an issue

**Comment by seligj95** [comment]
closing due to no response and inability to repro, please reopen if still an issue
